File: WebContent/page/app/model/entity/Supplier.js
/**
 * Created with JetBrains WebStorm.
 * User: kornicameister
 * Date: 04.11.12
 * Time: 01:25
 * To change this template use File | Settings | File Templates.
 */

Ext.define('WMS.model.entity.Supplier', {
    extend: 'WMS.model.entity.Client',
    proxy : {
        type       : 'wms',
        url        : 'wms/agent/client',
        extraParams: {
            type: 'supplier'
        }
    }
});

File: javaSrc/wms/model/Supplier.java
package wms.model;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;


@Entity
@DiscriminatorValue(value = "supplier")
public class Supplier extends Client {
    private static final long serialVersionUID = - 3487771558214472341L;

    public Supplier() {
        super();
    }

    @Override
    public String toString() {
        return "Supplier [toString()=" + super.toString() + "]";
    }

}
